legal-api/src/legal_api/core/filing.py

```diff
@@ -53,6 +53,12 @@ class Status(str, Enum):
         PAPER_ONLY = 'PAPER_ONLY'
         PENDING_CORRECTION = 'PENDING_CORRECTION'
 
+        # filings with staff review
+        APPROVED = 'APPROVED'
+        AWAITING_REVIEW = 'AWAITING_REVIEW'
+        CHANGE_REQUESTED = 'CHANGE_REQUESTED'
+        REJECTED = 'REJECTED'
```

Comment:
Are you sure about AWAITING_REVIEW?

Reply:
I did not understand

Reply:
The comment says, "filings with staff review". I am not exactly sure what this refers to. However, only Approved, Change Requested, and Rejected can have a review and comment, right?

Reply:
Oh! So a filing that is awaiting review also has a review object? In that case, yes, these 4 filing states have a review object. Resolving.